commands.py

```diff
@@ -1939,12 +1879,7 @@ def mycolour(message_dict):
 
 @is_command("Check all the crypto prices.")
 def crypto(message_dict):
-	user = message_dict["display-name"].lower()
-	message_dict["suppress_log"] = True
-	btc(message_dict)
-	eth(message_dict)
-	doge(message_dict)
-	log(f"Sent Crypto prices to {user}")
+	crypto_command.crypto(message_dict)

```

Comment:
I don't really see the reason to put this in a separate file - why don't you want it to stay in `commands.py`?

Reply:
Because I'm going to start parsing the commands out of `commands.py` as well. It's exceptionally unwieldy to maintain already and is still being added to. We're making it perfectional. ;)